.github/workflows/deploy.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     runs-on: ubuntu-latest
     name: deploy
     env:
-      gh_token: ${{ secrets.GH_TOKEN }}
```

Comment:
This secret could be removed for now in settings?

Reply:
Yes, thanks!